analysis/analysis.py

```diff
@@ -1,120 +1,15 @@
-"""
```

Comment:
This file was stripped and made into the parent Analysis class for the two child classes. The functionalities are put in those child classes.